requirements.txt

```diff
@@ -53,7 +53,7 @@ traceback2==1.4.0
 translationstring==1.3
 umemcache==1.6.3
 unittest2==1.1.0
-urllib3==1.24.2
+urllib3==1.25.9
```

Comment:
This is the one that failed the check (CVE-2020-26137) - `requests` had to be bumped up at the same time.